README.md

```diff
@@ -42,26 +42,26 @@ Please see
 
```

Comment:
Can you
* finish the sentence under the heading # The pain of cross-cluster Kubernetes?
* explain what backends are under # Kedge design